src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -127,7 +128,8 @@ def test_WINNF_FT_S_PAT_1(self, config_filename):
            passed_tests += this_test*1.0
        except AssertionError as e:
            # Allow HTTP status 400
-           if (refResponse == 400) and (e.args[0] == 400):
-               passed_tests += 1.0
+           self.assertEqual(e.args[0], refResponse)
```

Comment:
I still do not understand the use of the `try:  except:` clause : it tries to capture an AssertionError that no one raises.
Can you please explain ?